.NET/Microsoft.Recognizer.Text.TimeZone/Constants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Microsoft.Recognizers.Text.TimeZone
```

Comment:
Hi @jianke03, Timezone should not be a separate project. Timezones should be a model under DateTime as we want to use them in resolution without devs having to depend on two packages.